app/src/main/java/com/amaze/filemanager/activities/TextEditorActivity.java

```diff
@@ -296,19 +297,22 @@ private void load() {
                     try {
                         mInput.setText(data.fileContents);
                         if (data.fileContents.isEmpty()) {
-                            mInput.setHint(R.string.file_empty);
+                            Toast.makeText(getApplicationContext(), R.string.file_empty, Toast.LENGTH_SHORT).show();
```

Comment:
Revert to hint, "file is empty" is not a instantaneous error, it is a state the file is in, we are alerting the user via an always visible hint.